src/main/java/org/akaza/openclinica/service/PdfServiceImpl.java

```diff
@@ -260,94 +262,58 @@ public String preparePdfHeader(Study study, Study site, String studySubjectIdent
-            */
-           message = message.replace("\n", "").replace("\r", "");
-           
-           while (message.length() > 0)
-           {
-               int spaceIndex = message.indexOf(' ', lastSpace + 1);
-               if (spaceIndex < 0)
-                   spaceIndex = message.length();
-               String subString = message.substring(0, spaceIndex);
-               float size = subString.length();
-            
-               if (size > width)
-               {
-                   if (lastSpace < 0) {
-                	    lastSpace = spaceIndex;
-                   }
-                      
-                   subString = message.substring(0, lastSpace);
-                   lines.add(subString);
-                   message = message.substring(lastSpace).trim();
-                
-                   lastSpace = -1;
-               }
-               else if (spaceIndex == message.length())
-               {
-                   lines.add(message);                
-                   message = "";
-               }
-               else
-               {
-                   lastSpace = spaceIndex;
-               }
-           }
-
-           contents.beginText();
-           contents.setFont(pdfFont, fontSize);
-           contents.newLineAtOffset(startX,startY);
-           for (String line: lines)
-           {
-        	   contents.showText(line);
-        	   contents.newLineAtOffset(0, -leading);
-           }
-           contents.endText(); 
-           contents.close();
-          
-           
-           doc.save(fileName);
-       }catch(Exception e) {
-    	   logger.error("Error " + e.getMessage());
+       	 file.createNewFile();
+       	 writer = new PrintWriter(file.getPath(), "UTF-8");
+       	 String subStr = "";
+       	 
+       	 // write header in the UI table
+       	 writer.println("SubjectKey,ParticipantID,Status,Timestamp,Message"); 
+       	 
+       	 SimpleDateFormat formatter= new SimpleDateFormat("yyyy-MM-dd HH:mm:ss z");
+       	 Date date = new Date(System.currentTimeMillis());
+       
+       	 writer.println(ss.getOcOid()+","+ss.getLabel()+","+"Failed"+","+formatter.format(date)+","+"Error Detail:");
+       	 
+       	 //during testing, found double quote make JSP error
+       	 msg = msg.replaceAll("\"", "");
+       	 
+       	 if(msg.length() > 100) {
+       		 while(msg.length() > 100) {
+       			 int endIndex = msg.indexOf(" ", 100);
+       			 subStr =  msg.substring(0, endIndex);
+       			 writer.println(", , , ," + subStr); 
+       			 
+       			 msg = msg.substring(endIndex);
+       		 }
+       		// write the last line
+       		writer.println(", , , ," +msg);
+       		
+       	 }else {
+       		  writer.println(", , , ," +msg); 
+       	 }
+       	    
+       } catch (IOException e) {
+       	 logger.error("Error while accessing file to start writing: ",e);
+		} finally {                        
+           writer.close();;
```

Comment:
instead of wrapping the error message, put the entire message into the same column

Reply:
done